.github/workflows/self-comment-ci.yml

```diff
@@ -0,0 +1,251 @@
+name: PR comment GitHub CI
+
+on:
+  issue_comment:
+    types:
+      - created
+    branches-ignore:
+      - main
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.issue.number }}-${{ startsWith(github.event.comment.body, 'run-slow') }}
+  cancel-in-progress: true
```

Comment:
does it only cancel for the PR?

Reply:
it will ONLY cancel the one triggered by the same PR (or issue). The runs triggered by other PR/issue won't be canceled. 

  - (If it is an issue, it won't run any actual job: we restrict this below)